src/app/models/field/dateField.ts

```diff
@@ -23,6 +23,10 @@ const createDateFieldSchema = () => {
         default: null,
       },
     },
+    invalidDaysOfTheWeek: {
+      type: [Number],
+      default: [],
```

Comment:
nit: don't need default for arrays, since mongoose [adds an array by default](https://mongoosejs.com/docs/faq.html#array-defaults)

Reply:
Updated PR